src/pages/Home.js

```diff
@@ -62,26 +62,30 @@ export default function Home(props) {
 
   return (
     <main>
-      <h1>Welcome To Honey Do</h1>
-      <p>Your Smart Shopping List.</p>
+      <h1 className="mt-5 text-3xl text-center self-start font-light">
+        Welcome to Honey Do. Your smart shopping list.
+      </h1>
```

Comment:
The font looks so good! I think it would be helpful to add a `<span>` or to put the sentence "Your smart shopping list." in another element. Something to make the initial greeting of "Welcome to Honey Do." to stand out more when the user first arrives.

Reply:
I agree, we should guarantee that line break so this won't happen. I'm on it :D